crates/re_log_types/src/path/entity_path.rs

```diff
@@ -97,6 +97,16 @@ impl EntityPath {
         Self::from(parts)
     }
 
+    /// Treat the file path as one opaque string.
+    ///
+    /// The file path separators will NOT become splits in the new path.
+    /// The returned path will only have one part.
+    pub fn from_file_path(file_path: &std::path::Path) -> Self {
```

Comment:
Interesting. This is effectively a workaround for the fact that we don't yet have a way to specify "I want individual views for each of these", right?
If so I would add a `TODO: use blueprint instead` here (also: cc @jleibs).

The fact that this does the opposite of what one would naturally expect deserves a more alarming name though: `from_file_path_as_mono` or something?

Reply:
Good call on the name.

I think mapping file paths to entity path _could_ make sense, but it's not absolutely clear to me it is a good idea to map one hierarchy to the other. And yeah, it causes problems like they all sharing the same space view